### PR TITLE
Add config.yaml, rego related files, and update the README.md

### PR DESCRIPTION
**Pull request by nguyenhien1994**
For easier to understand, I would like to add the config.yaml, rego, data, example input files, and update the READMD.md.
It's really important for the beginners to understand this example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,4 +45,23 @@ allow {
...
     input.user == "bob"
 }
-```
\ No newline at end of file
+```
+
+## Trying the example
+- As a manager, create a car (this should be allowed):
+```
+curl -H 'Authorization: alice' -H 'Content-Type: application/json' \
```
Comment (anderseknert) [inline README.md:53]: `Authorization: alice` doesn't look right. You're probably looking to include the `user` as part of the input, as that is what your policy is looking for.

File: README.md [hunk 2]
```diff
@@ -45,4 +45,23 @@ allow {
...
+    -X PUT localhost:8080/cars/test-car \
+    -d '{"model": "Toyota", "vehicle_id": "357192", "owner_id": "4821", "id": "test-car"}'
+```
+
+- As a car admin, try to delete a car (this should be denied):
+```
+curl -H 'Authorization: kelly' \
```
Comment (anderseknert) [inline README.md:60]: Same here.

File: README.md [hunk 3]
```diff
@@ -45,4 +45,23 @@ allow {
...
+
+## Trying the example
+- As a manager, create a car (this should be allowed):
+```
+curl -H 'Authorization: alice' -H 'Content-Type: application/json' \
+    -X PUT localhost:8080/cars/test-car \
+    -d '{"model": "Toyota", "vehicle_id": "357192", "owner_id": "4821", "id": "test-car"}'
+```
+
+- As a car admin, try to delete a car (this should be denied):
```
Comment (anderseknert) [inline README.md:58]: Hmm, a "car admin" is a role I think is a little difficult to relate to, and why it wouldn't be allowed to DELETE cars. Could we come up with a better role name?

File: config.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+services:
+  play:
+    url: https://play.openpolicyagent.org
+    response_header_timeout_seconds: 5
+    credentials:
+      bearer:
+        token: "bGFza2RqZmxha3NkamZsa2Fqc2Rsa2ZqYWtsc2RqZmtramRmYWxkc2tm"
```
Comment (anderseknert) [inline config.yaml:7]: This should not be necessary for a playground policy.

File: config.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+labels:
+  app: myapp
+  region: west
+  environment: production
```
Comment (anderseknert) [inline config.yaml:12]: I don't think we should have these labels here.

File: config.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+bundles:
+  authz:
+    service: play
+    resource: bundles/WfbLy2DbiF
+    persist: true
+    polling:
+      min_delay_seconds: 60
+      max_delay_seconds: 120
+    signing:
```
Comment (anderseknert) [inline config.yaml:22]: I don't think this bundle is signed.

File: config.yaml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+      scope: write
+
+decision_logs:
+  console: true
+
+default_decision: /http/example/authz/allow
+
+persistence_directory: /var/opa
+
+keys:
```
Comment (anderseknert) [inline config.yaml:33]: As far as I can see, this is not needed.

File: config.yaml [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  global_key:
+    algorithm: RS256
+    key: <PEM_encoded_public_key>
+    scope: read
+
+caching:
```
Comment (anderseknert) [inline config.yaml:39]: This won't be needed here either.

File: rego/input.json [hunk 9]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (anderseknert) [inline rego/input.json:1]: I'm not sure how this relates to the input values provided in the example from the README.

File: rego/data.json [hunk 10]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (anderseknert) [inline rego/data.json:1]: How/where is this file used?

**Review by anderseknert — COMMENTED** [review]
Thanks for contributing to this project! I've left some comments on things to look into, and I think we could remove quite a bit from here in order to not complicate things more than they need to :) But I'm definitely in favor of providing at least examples on how to query the service.

File: README.md (hunk 1 above)
Comment (nguyenhien1994) [reply README.md:53]: Your code is using this way to include the user to input: https://github.com/open-policy-agent/example-api-authz-go/blob/2b56096cb0fa68f6550b66d0d68a0250ee904ebc/internal/api/api.go#L216

File: rego/data.json (hunk 10 above)
Comment (nguyenhien1994) [reply rego/data.json:1]: It's just the mockDB from your program. Maybe we don't need it, I will remove it.

File: rego/input.json (hunk 9 above)
Comment (nguyenhien1994) [reply rego/input.json:1]: Just the example input, anw, I will remove it.

File: README.md (hunk 3 above)
Comment (nguyenhien1994) [reply README.md:58]: I took it from there https://github.com/open-policy-agent/example-api-authz-python

File: config.yaml (hunk 8 above)
Comment (nguyenhien1994) [reply config.yaml]: remove unused parts

File: README.md (hunk 1 above)
Comment (anderseknert) [reply README.md:53]: Oh, I should probably have checked the code for that handler 😄 That's really not a nice way to use the HTTP spec, but I guess we can live with that for now then.

File: config.yaml [hunk 11]
```diff
@@ -0,0 +1,20 @@
+services:
```
Comment (anderseknert) [inline config.yaml:1]: This looks better! I would probably aim for making this config file as minimal as possible though, as to not distract from the example. Would something like this be enough?

```yaml
services:
  play:
    url: https://play.openpolicyagent.org

bundles:
  authz:
    service: play
    resource: bundles/WfbLy2DbiF

decision_logs:
  console: true

default_decision: /http/example/authz/allow
```

File: README.md (hunk 3 above)
Comment (anderseknert) [reply README.md:58]: OK. I'd probably prefer that to be changed too, to be honest. But let's keep the policy as it is for now then.

**Review by anderseknert — APPROVED** [review]
LGTM. Perhaps @ashutosh-narkar can weigh in on whether we should use Playground links in examples like this.

File: config.yaml (hunk 11 above)
Comment (nguyenhien1994) [reply config.yaml:1]: Yes, I just updated the config as your suggestion.

File: config.yaml [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+
+bundles:
+  authz:
+    service: play
+    resource: bundles/WfbLy2DbiF
+
+decision_logs:
+  console: true
+
+default_decision: /http/example/authz/allow
```
Comment (ashutosh-narkar) [inline config.yaml:13]: Is this file used anywhere ? If yes then where is the policy corresponding to the  `default_decision` ?

**Comment by ashutosh-narkar** [comment]
> LGTM. Perhaps @ashutosh-narkar can weigh in on whether we should use Playground links in examples like this.

I would prefer to use a local Nginx instance instead. @nguyenhien1994 we updated the [Envoy](https://www.openpolicyagent.org/docs/edge/envoy-tutorial-standalone-envoy/#4-publish-opa-bundle) tutorial recently to use Nginx as the bundle server if you're looking for a reference.

**Comment by anderseknert** [comment]
Closing due to inactivity. Let me know if you want to revisit this later @nguyenhien1994

**Pull request closed without merging**